constrain/demo/tspr_cases/tspr_cases.py

```diff
@@ -0,0 +1,166 @@
+"""The goal of this demo cases is to showcase how verification cases can be semi-automatically created for building energy simulation timeseries.
+"""
+
+import matplotlib.pyplot as plt
+import json, glob
+
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+
+import constrain as cs
+from constrain.workflowsteps import *
+from constrain.library import *
+from constrain.libcases import *
+
+# Input/Output params
+# Change the following according to your system configuration
+energy_plus_simulation = False
+energy_plus_path = "/Applications/EnergyPlus-25-1-0/energyplus"
+energy_plus_idd = "../../../resources/Energy+V25_1_0.idd"
+energy_plus_weather_file = (
+    "/Applications/EnergyPlus-25-1-0/WeatherData/USA_CO_Golden-NREL.724666_TMY3.epw"
+)
```

Comment:
These need to be updated locally so EnergyPlus can run. It would be nice to have the internal EP runner rely on Docker so we could run these simulations through GitHub actions.